### ISSUE TITLE
Do you implement the P3D structure?

### ISSUE THREAD

**Issue by Practice-Machines** [MRzzm/action-recognition-models-pytorch#1]
Have you implemented the P3D structure? When I was researching this network, I couldn't reach the accuracy mentioned in the article. I don't know if there is a problem with the pre-training data set or what optimization problem? Do you have research in this area? Hope to receive your reply.

**Comment by MRzzm** [comment]
Hi，JianyuChenn.  I have not trained p3d model  from scratch on ucf101 or other dataset.  If you just want to take p3d as one feature extractor, it would be better to download the kinetics pre-trained weight and finetune on your dataset. If you follow the origin paper, we can further disguss the training details in email.

**Comment by MRzzm** [comment]
Hi，JianyuChenn. I have not trained p3d model from scratch on ucf101 or other dataset. If you just want to take p3d as one feature extractor, it would be better to download the kinetics pre-trained weight and finetune on your dataset. If you follow the origin paper, we can further disguss the training details in email.

> Have you implemented the P3D structure? When I was researching this network, I couldn't reach the accuracy mentioned in the article. I don't know if there is a problem with the pre-training data set or what optimization problem? Do you have research in this area? Hope to receive your reply.

Hi，JianyuChenn.  I have not trained p3d model  from scratch on ucf101 or other dataset.  If you just want to take p3d as one feature extractor, it would be better to download the kinetics pre-trained weight and finetune on your dataset. If you follow the origin paper, we can further disguss the training details in email.

**Comment by MRzzm** [comment]
> Have you implemented the P3D structure? When I was researching this network, I couldn't reach the accuracy mentioned in the article. I don't know if there is a problem with the pre-training data set or what optimization problem? Do you have research in this area? Hope to receive your reply.

Hi，JianyuChenn. I have not trained p3d model from scratch on ucf101 or other dataset. If you just want to take p3d as one feature extractor, it would be better to download the kinetics pre-trained weight and finetune on your dataset. If you follow the origin paper, we can further disguss the training details in email.
